src/main/resources/templates/discover.html

```diff
@@ -4,10 +4,11 @@
 <body>
 <header th:replace="fragments/layout :: header"> </header>
 
-<section th:each=" user: ${users)">
+<section th:each=" user: ${users}">

```

Comment:
This is a good start, should we fragmentize this into a `fragments/user.html` file?

Reply:
Maybe in a future PR